pyro/contrib/minipyro.py

```diff
@@ -275,14 +264,14 @@ def __call__(self, params):
 
 
 # This is a unified interface for stochastic variational inference in Pyro.
-# The actual construction of the loss is taken care of by `loss`.
+# The actual construction of the loss is taken care of by `loss_factory`.
 # See http://docs.pyro.ai/en/stable/inference_algos.html
 class SVI:
-    def __init__(self, model, guide, optim, loss):
+    def __init__(self, model, guide, optim, loss_factory):
         self.model = model
         self.guide = guide
         self.optim = optim
-        self.loss = loss
+        self.loss_factory = loss_factory
```

Comment:
:+1: I think it's safe to rename the member. Is that what the main-Pyro SVI class does, so users won't be confused?